.gitignore

```diff
@@ -4,6 +4,7 @@ log*.html
 *.xcodeproj
 .vscode
 .cache
+.zig-cache
```

Comment:
```suggestion
zig-cache
```

Pretty sure it's just `zig-cache`, no dot.

Reply:
Please refer to the PR description; `zig-cache` is already covered in the `.gitignore` by `zig-*`. This is an addition to handle the new rename to `.zig-cache`.

https://github.com/ziglang/zig/pull/20115

Reply:
sorry, did not read the PR desc, my bad